Lib/test/test_future.py

```diff
@@ -290,6 +284,15 @@ def test_annotations(self):
         eq("(x:=10)")
         eq("f'{(x:=10):=10}'")
 
+        # f-strings with '=' don't round trip very well, so set the expected
```

Comment:
I think it is worth to move this into separate method. It can even be decorated as `cpython_only`. I think that if an alternate implementation is smart enough to roundtrip these case, it is still a valid Python implementation.